services/core/resources/constructs/review-bucket.ts

```diff
@@ -11,7 +11,7 @@ export class ReviewBucket extends Bucket {
       removalPolicy: isProduction()
         ? RemovalPolicy.RETAIN
         : RemovalPolicy.DESTROY,
-      autoDeleteObjects: !isProduction(),
+      // autoDeleteObjects: !isProduction(),
```

Comment:
@mattzcarey Seb and I chatted about this. Adding this autoDeleteObjects thing is a bit broken, and it doesn't allow you to tear down your stack (you will see a couple  of stacks already have DeleteFailed). So we have both commented this out for now. Might be a good things to investigate!

Reply:
Sounds good!